src/olympia/translations/widgets.py

```diff
@@ -65,7 +67,34 @@ def render(self, name, value, attrs=None):
             default_locale = getattr(self, 'default_locale',
                                      translation.get_language())
             value = [Translation(locale=default_locale)]
-        return super(TransMulti, self).render(name, value, attrs)
+
+        if self.is_localized:
+            for widget in self.widgets:
+                widget.is_localized = self.is_localized
+
+        # value is a list of values, each corresponding to a widget
+        # in self.widgets.
+        if not isinstance(value, list):
+            value = self.decompress(value)
+
+        output = []
+
+        if attrs is None:
+            attrs = {}
+
+        final_attrs = self.build_attrs(attrs)
+        id_ = final_attrs.get('id', None)
+        for i, widget in enumerate(self.widgets):
+            try:
+                widget_value = value[i]
+            except IndexError:
+                widget_value = None
+            if id_:
+                final_attrs = dict(final_attrs, id='%s_%s' % (id_, i))
+            output.append(widget.render(
+                name + '_%s' % i, widget_value, final_attrs
+            ))
+        return mark_safe(self.format_output(output))
```

Comment:
This is just forward-ported from 1.8 to keep compatibility. Needs form-refactor for using templates in the future.